.github/workflows/build.yml

```diff
@@ -58,12 +58,10 @@ jobs:
           ORG_GRADLE_PROJECT_SONATYPE_NEXUS_USERNAME: ${{ secrets.SONATYPE_NEXUS_USERNAME }}
           ORG_GRADLE_PROJECT_SONATYPE_NEXUS_PASSWORD: ${{ secrets.SONATYPE_NEXUS_PASSWORD }}
 
-      # TODO(egorand): Publish website
-      #
-      # - name: Prep mkdocs
-      #   run: .buildscript/prepare_mkdocs.sh
-      #
-      # - name: Deploy docs
-      #   uses: mhausenblas/mkdocs-deploy-gh-pages@master
-      #   env:
-      #     PERSONAL_TOKEN: ${{ secrets.PERSONAL_TOKEN }}
+      - name: Prep mkdocs
+        run: .buildscript/prepare_mkdocs.sh
+
+      - name: Deploy docs
+        uses: mhausenblas/mkdocs-deploy-gh-pages@master
```

Comment:
Should we do this only on tags? If I add a new feature and document it it'll show up on the website before it's released. The counterargument, of course, is that if you fix something you have to wait until the next release before it's deployed (or do it manually).

Reply:
Publishing on tags makes sense, although as Kotlinpoet is mostly feature-complete and we're mostly fixing things rather than adding new APIs, I think we'll have more instances of forgetting to publish the website than people complaining about unreleased APIs showing up in the docs. I'm happy with either though.

Reply:
Let's just merge as-is and we can change if needed.